addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2849,7 +2849,9 @@ export class OdooEditor extends EventTarget {
         }
 
         // handle stars
-        if (node.nodeType === Node.ELEMENT_NODE && node.className.includes('fa-star') &&
+        if (node.nodeType === Node.ELEMENT_NODE &&
+            node.className.includes && // beware of SVG's className
+            node.className.includes('fa-star') &&
```

Comment:
Indeed, your fix works :+1: . As you say in the commit message, `.className` on a SVG HTML element is not a string (like on other elements) but `.classList` always returns a `DomTokenList` so personally I would find it better if instead of what you do you replace the `node.className.includes('XXX')` by `node.classList.contains('XXX')`

Reply:
Well... the current `className.includes()` is a bit abusive: it matches both `fa-star` and `fa-star-o` (the outlined star icon).
I should either add a comment about this or just match both specific class names through `classList`.